services/docker-compose.yml

```diff
@@ -249,18 +249,6 @@ services:
       - ../.env
       - ../.env-wb-garbage-collector
 
-    healthcheck:
```

Comment:
not sure I understand. from the PR you wrote that you added a /health entrypoint. why do you remove the healthcheck here?

Reply:
@sanderegg The healthcheck defined in the Dockerfile makes a requires to ``v0/health`` but that entrypoint is only created if the ``diagnostics`` plugin is active, which is not the case with the ``wb_garbage_collector`` setup.

In the last PR, the solution I adopted was overriding the healtcheck on the docker-compose and redirecting to the entrypoint ``/`` which is created by ``rest`` plugin.  
But there was a problem I did not notice then: when you use the stack in development mode all service healthchecks were disabled except for the ``wb_garbage_collector`` service ... and since the timings were set for production, the swarm would never start that service.

Therefore, instead of "polluting" more  the docker-compose files, I found that it was simpler to recreate ``v0/health`` in the ``rest`` plugin when the ``diagnostics`` plugin was disabled.